locale/de/foreman_puppet.po

```diff
@@ -68,10 +68,10 @@ msgid ""
 "        <b>Matcher:</b> %{matcher}<br/>\n"
 "        <b>Inherited value:</b> %{inherited_value}"
 msgstr ""
-"<b>Beschreibung</b>: %{desc}<br/>\n"
-"<b>Typ</b>: %{Typ} <br/>\n"
-"<b> Matcher</b>: %{matcher}<br/>\n"
-"<b>Geerbter Wert:</b> %{inherited_value}"
+"<b>Beschreibung:</b> %{desc}<br/>\n"
+"     <b>Typ:</b> %{type}<br/>\n"
```

Comment:
this is prudent to get in as it causes https://projects.theforeman.org/issues/34400